src/map/ai/helpers/targetfind.cpp

```diff
@@ -317,15 +329,15 @@ bool CTargetFind::isMobOwner(CBattleEntity* PTarget)
     return found;
 }
 
-/*
-validEntity will check if the given entity can be targeted in the AoE.
+// validEntity will check if the given entity can be targeted in the AoE.
```

Comment:
I agree with your change here, but just want to point out there will be some instances where a block comment will be proffered - for really big long comments like our copyright notice. So we're unfortunately never going to have a single comment style. some inline, some block, some single line like this. As much as I am OCD about being consistent, they have their uses. I wish we had it codified in the style guide when to do which. I very much welcome discussion followed by PR's about said style guide, I never finished it.

Reply:
Thanks for the help. I am new to this project and trying to learn what is and isn't part of the style guide. So I will take that into consideration in the future. Of course, I doubt any of this will be merged, but in the very least it's helpful to have yours and others' comments in order to help me match the agreed style.

The brackets and such are most likely just people's IDEs or text editors set in a mode from working on other projects. I wish I was more familiar with this project because I haven't been able to fix any actual code at this point, simply esoteric nonsense and at some points blatantly wrong. I apologize, and I hope to improve with time as I read through the code. 

This is basically me reading through the code lol

Reply:
I apologize that our guide is missing a lot :x I'll try and get more done with that soonish, maybe you can leave me some nice input for what needs to be there :)

I get distracted a lot by the different styles/lack of styling in some files. especially when I see 3 or more styles in the same file. I like things to conform to one way as much as possible and don't even care which way it is and that's made working in some files a struggle for me. I appreciate what you are attempting to do with this pull request, some of it might not get merged but with whatever changes you get asked to do the rest could its not an all or nothing deal.